tools/test_cases.F90

```diff
@@ -5566,7 +5570,8 @@ end subroutine superK_u
 
 
  subroutine SuperCell_Sounding(km, ps, pk1, tp, qp)
- use gfdl_cloud_microphys_mod, only: wqsat_moist, qsmith_init, qs_blend
+ use gfdl_cloud_microphys_mod, only: qsmith_init, qs_blend
+ use gfdl_mp_mod, only: wqsat_moist
```

Comment:
I think we need to be careful about which one we use.  As I understand it, one is to be compiled with -r8 (physics) and the other with -r4 (inline dycore physics) - but maybe I've got it all wrong.